keyboards/enviousdesign/65m/info.json

```
@@ -0,0 +1,103 @@
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": ["GP0", "GP1", "GP2", "GP3", "GP4", "GP5", "GP6", "GP7", "GP8", "GP9", "GP10", "GP11", "GP12", "GP13", "GP14", "GP15", "GP16"],
+        "rows": ["GP21", "GP20", "GP19", "GP18", "GP17"]
+    },
+    "processor": "RP2040",
+    "url": "http://envious.design",
+    "usb": {
+        "device_version": "0.0.1",
+        "pid": "0xA279",
+        "vid": "0xE739"
+    },
+    "layouts": {
+        "LAYOUT": {
+            "layout": [
+                {"matrix": [0, 0], "x": 0, "y": 0},
+                {"matrix": [0, 1], "x": 1, "y": 0},
+                {"matrix": [0, 2], "x": 2, "y": 0},
+                {"matrix": [0, 3], "x": 3, "y": 0},
+                {"matrix": [0, 4], "x": 4, "y": 0},
+                {"matrix": [0, 5], "x": 5, "y": 0},
+                {"matrix": [0, 6], "x": 6, "y": 0},
+                {"matrix": [0, 7], "x": 7, "y": 0},
+                {"matrix": [0, 8], "x": 8, "y": 0},
+                {"matrix": [0, 9], "x": 9, "y": 0},
+                {"matrix": [0, 10], "x": 10, "y": 0},
+                {"matrix": [0, 11], "x": 11, "y": 0},
+                {"matrix": [0, 12], "x": 12, "y": 0},
+                {"matrix": [0, 13], "x": 13, "y": 0, "w": 2},
+                {"matrix": [1, 0], "x": 0, "y": 1, "w": 1.5},
+                {"matrix": [1, 2], "x": 1.5, "y": 1},
+                {"matrix": [1, 3], "x": 2.5, "y": 1},
+                {"matrix": [1, 4], "x": 3.5, "y": 1},
+                {"matrix": [1, 5], "x": 4.5, "y": 1},
+                {"matrix": [1, 6], "x": 5.5, "y": 1},
+                {"matrix": [1, 7], "x": 6.5, "y": 1},
+                {"matrix": [1, 8], "x": 7.5, "y": 1},
+                {"matrix": [1, 9], "x": 8.5, "y": 1},
+                {"matrix": [1, 10], "x": 9.5, "y": 1},
+                {"matrix": [1, 11], "x": 10.5, "y": 1},
+                {"matrix": [1, 12], "x": 11.5, "y": 1},
+                {"matrix": [1, 13], "x": 12.5, "y": 1},
+                {"matrix": [2, 14], "x": 13.75, "y": 1, "w": 1.25, "h": 2},
+                {"matrix": [1, 14], "x": 18, "y": 1, "w": 1.5},
+                {"matrix": [2, 0], "x": 0, "y": 2, "w": 1.75},
+                {"matrix": [2, 2], "x": 1.75, "y": 2},
+                {"matrix": [2, 3], "x": 2.75, "y": 2},
+                {"matrix": [2, 4], "x": 3.75, "y": 2},
+                {"matrix": [2, 5], "x": 4.75, "y": 2},
+                {"matrix": [2, 6], "x": 5.75, "y": 2},
+                {"matrix": [2, 7], "x": 6.75, "y": 2},
+                {"matrix": [2, 8], "x": 7.75, "y": 2},
+                {"matrix": [2, 9], "x": 8.75, "y": 2},
+                {"matrix": [2, 10], "x": 9.75, "y": 2},
+                {"matrix": [2, 11], "x": 10.75, "y": 2},
+                {"matrix": [2, 12], "x": 11.75, "y": 2},
+                {"matrix": [2, 13], "x": 12.75, "y": 2},
```

Matches this data to the keymaps.

```suggestion
                {"matrix": [1, 14], "x": 13.5, "y": 1, "w": 1.5},
                {"matrix": [2, 0], "x": 0, "y": 2, "w": 1.75},
                {"matrix": [2, 2], "x": 1.75, "y": 2},
                {"matrix": [2, 3], "x": 2.75, "y": 2},
                {"matrix": [2, 4], "x": 3.75, "y": 2},
                {"matrix": [2, 5], "x": 4.75, "y": 2},
                {"matrix": [2, 6], "x": 5.75, "y": 2},
                {"matrix": [2, 7], "x": 6.75, "y": 2},
                {"matrix": [2, 8], "x": 7.75, "y": 2},
                {"matrix": [2, 9], "x": 8.75, "y": 2},
                {"matrix": [2, 10], "x": 9.75, "y": 2},
                {"matrix": [2, 11], "x": 10.75, "y": 2},
                {"matrix": [2, 12], "x": 11.75, "y": 2},
                {"matrix": [2, 13], "x": 12.75, "y": 2},
                {"matrix": [2, 14], "x": 13.75, "y": 2, "w": 1.25},
```